src/ssl.c

```diff
@@ -17086,10 +17197,38 @@ int wolfSSL_set_min_proto_version(WOLFSSL* ssl, int ver)
     return CheckSslMethodVersion(ssl->version.major, ssl->options.mask);
 }
 
-int wolfSSL_set_max_proto_version(WOLFSSL* ssl, int ver)
+int wolfSSL_set_min_proto_version(WOLFSSL* ssl, int version)
 {
+    const int verTbl[] = {SSL3_VERSION, TLS1_VERSION, TLS1_1_VERSION,
+                          TLS1_2_VERSION, TLS1_3_VERSION,DTLS1_VERSION,
+                          DTLS1_2_VERSION};
+    int tblSz = sizeof(verTbl);
```

Comment:
tblSz is going to end up `NUMBER_OF_PROTOCOLS * sizeof(int)`. The loop later is going to run off the end of verTbl.

Reply:
Changed to use NUMBER_OF_PROTOCOLS as the number of protocol  instead of tblSz.